Library/Formula/cvc4.rb

```diff
@@ -0,0 +1,29 @@
+class Cvc4 < Formula
+  homepage "http://cvc4.cs.nyu.edu/"
+  url "http://cvc4.cs.nyu.edu/builds/src/cvc4-1.4.tar.gz"
+
+  devel do
+    url "http://cvc4.cs.nyu.edu/builds/src/unstable/latest-unstable.tar.gz"
+    version "1.5-prerelease"
+  end
+
+  depends_on "boost" => :build
+  depends_on "gmp"
+  depends_on "libantlr3c" => ["with-exceptions", "without-shared"]
+
+  def install
+    args = ["--enable-static",
+            "--enable-shared",
+            "--with-compat",
+            "--bsd",
+            "--with-gmp",
+            "--with-antlr-dir=#{Formula["libantlr3c"].opt_prefix}",
+            "--prefix=#{prefix}"]
+    system "./configure", *args
+    system "make", "install"
+  end
+
+  test do
+    system bin/"cvc4", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!